### PR TITLE
Update voice_registry_system.md - issue 86

### PR DESCRIPTION
**Pull request by rogerkibbe**
Document implicit vs explicit invocation and voice assistant system use of VRS

### CODE REVIEW

File: components/voice_registry_system.md [hunk 1]
```diff
@@ -246,7 +246,31 @@ By following the SRP, we achieve the following: <br>
 #### 1.0.6.1 Do we need central location for common words?
 //todo: #85
 #### 1.0.6.2 Who is the decision maker whether user's utterance is explicit or implicit invocation?
-//todo: #86
+The voice assistant system will decide if a user's utterance is explicit or implicit. For the initial release, VRS will only handle explicit invocations and the voice assistant system will disambiguate implicit invocations.  
+
+**Explicit Invocation Definition:** An invocation of a voice application where the name of the application is part of the invocation utterance. For example:
+- ```Ask Target, to add milk to my shopping list```
```
Comment (maridob) [inline components/voice_registry_system.md:252]: Per @Jim1151 , add some statement around different from wake word.

File: components/voice_registry_system.md [hunk 2]
```diff
@@ -246,7 +246,31 @@ By following the SRP, we achieve the following: <br>
...
 #### 1.0.6.2 Who is the decision maker whether user's utterance is explicit or implicit invocation?
-//todo: #86
+The voice assistant system will decide if a user's utterance is explicit or implicit. For the initial release, VRS will only handle explicit invocations and the voice assistant system will disambiguate implicit invocations.  
+
+**Explicit Invocation Definition:** An invocation of a voice application where the name of the application is part of the invocation utterance. For example:
+- ```Ask Target, to add milk to my shopping list```  
+- ```Open Target```  
+- ```Talk to Target```  
+- ```Ask Target, what’s on sale```  
+- ```Add milk to my Target shopping list```
```
Comment (maridob) [inline components/voice_registry_system.md:256]: Per @jcstine 's comment, add some examples around John's bakery and an explanation on disambiguation in an explicit request.

**Review by maridob — CHANGES_REQUESTED** [review]
Based on the meeting 06.24.2021

File: components/voice_registry_system.md (hunk 1 above)
Comment (rogerkibbe) [reply components/voice_registry_system.md:256]: Updated to include a section defining a wake word and VRS scope:

Note: The name of a voice application is different than the wake word. A wake word is the user's word(s) used to get a voice assistant to start listening. For instance, "Alexa", "Hey Google" and "Hi Bixby" are wake words. If a user says:
``` Alexa, ask Target to add milk to my shopping list ``` then  
``` Alexa``` is the wake word and ```Target``` is the name of the voice application

File: components/voice_registry_system.md (hunk 2 above)
Comment (rogerkibbe) [reply components/voice_registry_system.md:260]: Updated to include a John's Bakery example:

There will be instances where there is more than one voice application with the same name. For example:
- ```Ask John's Bakery, when do you open```
  
There may be more than one voice application named *John's Bakery*. Indeed, there may be many different *John's Bakery* voice applications. If a user's explicit invocation results in multiple VRS matches, VRS will return all of the matching records. Included in the records will be the address and other metadata about the voice applications. The voice assistant will then use this information and other user information to disambiguate. Possible ways to disambiguate the above example include:
- Using the address to find the nearest *John's Bakery*
- Using the last launched *John's Bakery* application
- Asking the user which *John's Bakery* to use (and likely store this in user preferences for future use)
- Some other AI based algorithm to find the best option for the user

**Review by rogerkibbe — COMMENTED** [review]
Requested changes made

**Comment by JohnTrammellTarget** [comment]
Fixes #86

**Comment by maridob** [comment]
Based on the meeting today, we will close this PR if there are not other feedback by end of today.

**Review by JohnTrammellTarget — APPROVED** [review]
+1

**Review by JohnTrammellTarget — APPROVED** [review]

**Review by JohnTrammellTarget — APPROVED** [review]

**Comment by rogerkibbe** [comment]
@maridob - Can you approve the changes you requested so I can merge. Tjhanks

**Review by maridob — APPROVED** [review]
Merge please!

**Comment by maridob** [comment]
@rogerkibbe - you can now merge!

**Pull request merged**